src/modules/mqtt/doc/mqtt_admin.xml

```diff
@@ -324,6 +324,27 @@ modparam("mqtt", "ca_path", "/etc/ssl/certs")
 		<programlisting format="linespecific">
 ...
 modparam("mqtt", "tls_method", "tlsv1.3")
+...
+		</programlisting>
+		</example>
+	</section>
+	<section id="mqtt.p.tls_alpn">
+		<title><varname>tls_alpn</varname> (str)</title>
+		<para>
+        Used to set the TLS ALPN option for cases where one TLS port is used to share multiple services. Prominent
+        example is AWS IoT, where you have to set the <varname>tls_alpn</varname> to "mqtt" to be able to connect
+        via MQTT.
+		</para>
+		<para>
+		<emphasis>
+			Default value is NULL, where no ALPN is set on the TLS connection.
```

Comment:
You can't set NULL in a mod-adam. I would say that "Default value is empty string, which means that no TLS ALPN is used.

Reply:
Yes, I think we have to change the others. NULL is in the code, not in the configuration.

Reply:
I'd propose to do this in a second step to not mix commits. So I'm going to amend the current one, and in a second pull request change the wording of all mod params regarding NULL

Reply:
Yes, lets not mix this PR with other enhancements.

Reply:
Fully agree, my intention was to change the others separately.

Reply:
Is there actually good practice on how to document a param which has no default value (aka is NULL)? I'm finding quite some old modules having it documented like this, see https://www.kamailio.org/docs/modules/devel/modules/auth_db.html#auth_db.p.load_credentials for example.

Reply:
It's a bit of a rabbit hole. https://www.kamailio.org/docs/modules/devel/modules/dispatcher.html#dispatcher.p.db_url is one of the most famous ones I guess. Any good example how to properly document a non-set behavior, so it's consistent?

Reply:
That's a bad one with "NULL" in quotes like it's a settable value. Let's focus now on make this new option better, and then we'll work on improving the existing docs to something non-developers may understand :-)

Reply:
Alright. So, I will keep the docs for this commit as-is with the NULL, so it's consistent with the documentation of the other params in this module. In parallel, I've prepared another PR on top of this one, which changes the docs of all module parameters at once, changing the wording from NULL to `Default value is that option is not set ([description of implication this has, e.g. random id or no auth or whatever])`. Agreed?

Reply:
Thank you for doing that. Agreed!